nubis/puppet/files/nubis_users/nubis-users.toml

```diff
@@ -0,0 +1,12 @@
+[template]
+src = "nubis-users.tmpl"
+dest = "/etc/puppet/yaml/nubis_users/common.yaml"
+prefix = "/environment/nubis-users"
```

Comment:
Could you make that prefix something more consistent with everything else, like : 
  /environments/%%ENVIRONMENT%%/global/user-managment

Otherwise, looking great!